po: Add Portuguese translation

skitt opened the pull request:
This was provided a while ago on https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=761589; it’s presumably missing translations for newer messages.

pfps commented:
It's definitely better than nothing, but it needs some work.

Can you change the commit message to "po:  Add Portuguese translation" and figure out what is wrong with the end of the file?  The style checker didn't like it.

Alternatively, if you are willing I can copy the file and fix it up locally.

skitt commented:
> Can you change the commit message to "po: Add Portuguese translation" and figure out what is wrong with the end of the file? The style checker didn't like it.

I installed `pre-commit` and fixed it up locally, and updated the commit message, thanks!
